packages/nim/build.sh

```diff
@@ -0,0 +1,55 @@
+TERMUX_PKG_HOMEPAGE=https://nim-lang.org/
+TERMUX_PKG_DESCRIPTION="Nim is a systems and applications programming language. Statically typed and compiled, it provides unparalleled performance in an elegant package."
+TERMUX_PKG_VERSION=0.17.2
+TERMUX_PKG_SRCURL=https://nim-lang.org/download/nim-$TERMUX_PKG_VERSION.tar.xz
+TERMUX_PKG_SHA256=aaff1b5023fc4a5708f1d7d9fd8e2a29f1a7f58bf496532ff1e9d7e7c7ec82bd
+TERMUX_PKG_HOSTBUILD=yes
+TERMUX_PKG_BUILD_IN_SRC=yes
+#TERMUX_PKG_BLACKLISTED_ARCHES="x86_64"
+TERMUX_PKG_DEPENDS="git, clang, libandroid-glob"
```

Comment:
These are runtime dependencies, why would it need git?

Reply:
its not a run time dependncy for nim its a dependncy for nimble and koch